### PR TITLE
Document duplication in Consul

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: plugins/anchor-links/index.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ const stringify = require('remark-stringify')
 
 // This plugin adds anchor links to headlines and lists that begin with inline
 // code blocks.
+//
+// NOTE: Some of the HTML code is duplicated in:
+//       https://github.com/hashicorp/consul/blob/517cfa79184dc2e43bd61007d4d76946f08591d3/website/components/config-entry-reference/index.jsx#L118-L136
```
Comment (lkysow) [inline plugins/anchor-links/index.js:11]: Will update with right link once that PR is merged.

Comment (jescalan) [reply plugins/anchor-links/index.js:11]: I'll hold off until after merge to run this one then? Or do you have any opinion on the ordering?

Comment (lkysow) [reply plugins/anchor-links/index.js:11]: Yeah hold off for the merge until I can get the permalink from the master branch.

Comment (lkysow) [reply plugins/anchor-links/index.js]: @jescalan this is ready to merge. I've updated the permalink.

**Pull request merged**